_posts/2025-01-16-19.0.0-release.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: post
+title: "Apache Arrow 19.0.0 Release"
+date: "2025-01-16 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 19.0.0 release. This release
+covers over 2 months of development work and includes [**202 resolved
+issues**][1] on [**330 distinct commits**][2] from [**67 distinct
+contributors**][2]. See the [Install Page](https://arrow.apache.org/install/) to
+learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 18.1.0 release, Adam Reeve and Laurent Goujon have been invited to
+become committers. Gang Wu has been invited to join the Project Management
+Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Release Highlights
+
+A [bug](https://github.com/apache/arrow/issues/45283) has been identified in the
+19.0.0 versions of the C++ and Python libraries which prevents reading Parquet
+files written by Arrow Rust v53.0.0 or higher. The files written by Arrow Rust
+are correct and the bug was in the patch adding support for Parquet's new
+[SizeStatistics](https://github.com/apache/parquet-format/pull/197) feature to
+Arrow C++ and Python. See [#45293](https://github.com/apache/arrow/issues/45283)
+for more details including a potential workaround.
+
+As a result, we plan to create a 19.0.1 release to include a fix for this which
+should be available in next few weeks.
```

Comment:
@pitrou @raulcd @kou can I get a +1 on this new paragraph before merging? With my text I was trying to make it clear where the issue lies and what the plan is.

Reply:
+1

Reply:
For those I pinged, I'm going to merge this but feel free to suggest changes if you have them and I can do a follow-up PR.